### PR TITLE
Graduate BlockVolume feature to beta

### PR DESCRIPTION
**Pull request by wongma7**
docs for feature https://github.com/kubernetes/features/issues/351

### CODE REVIEW

**Comment by zparnold** [comment]
@wongma7 Now that this is in beta, does it need more documentation surrounding it?

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ different Kubernetes components.
 | `APIResponseCompression` | `false` | Alpha | 1.7 | |
 | `AppArmor` | `true` | Beta | 1.4 | |
 | `AttachVolumeLimit` | `false` | Alpha | 1.11 | |
-| `BlockVolume` | `false` | Alpha | 1.9 | |
```
Comment (xiangpengzhao) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md]: We should keep this line just like other features :)

**Comment by justaugustus** [comment]
@xiangpengzhao @zhangxiaoyu-zidif -- what's the status of this? These docs are overdue and we haven't heard any status updates on the associated feature issue.

I'm removing this from the milestone per this [comment](https://github.com/kubernetes/features/issues/351#issuecomment-420196600).

If you feel this is incorrect, please respond ASAP.

/milestone clear

**Comment by xiangpengzhao** [comment]
@wongma7 please confirm @justaugustus concern :)

**Comment by tfogo** [comment]
Hi @wongma7, can you please complete this PR and let us know when it's ready for review? The docs deadline is approaching fast so we want to start reviewing PRs ASAP.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (wongma7) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: fixed !

**Comment by wongma7** [comment]
oops i rebased on master instead of dev zzzz

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -300,11 +300,12 @@ Currently, storage size is the only resource that can be set or requested.  Futu
 ### Volume Mode
 
 {{< feature-state for_k8s_version="v1.9" state="alpha" >}}
```
Comment (msau42) [inline content/en/docs/concepts/storage/persistent-volumes.md]: question for docs: do we still need the alpha info?

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 3]
```diff
@@ -616,19 +617,10 @@ Volume binding matrix for statically provisioned volumes:
 
 | PV volumeMode | PVC volumeMode  | Result           |
```
Comment (msau42) [inline content/en/docs/concepts/storage/persistent-volumes.md]: Can we simplify this table to just say volumeMode on PVC and PV must match?

Comment (wongma7) [reply content/en/docs/concepts/storage/persistent-volumes.md]: reduced to "`PersistentVolumeClaims` will only be bound to `PersistentVolumes` with the same `volumeMode.`" https://github.com/kubernetes/website/pull/10100/commits/c90a2a861cbe81ba9100a3fd20c16d91d6cb18eb#diff-bd23e5d028a7006d4c9e9cd8db0d5eabR614

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 4]
```diff
@@ -610,25 +611,7 @@ spec:
...
-|   Block       | Block           | BIND             |
-|   Block       | Filesystem      | NO BIND          |
-|   Filesystem  | Filesystem      | BIND             |
-|   Filesystem  | Block           | NO BIND          |
-|   Filesystem  | unspecified     | BIND             |
-
-{{< note >}}
-**Note:** Only statically provisioned volumes are supported for alpha release. Administrators should take care to consider these values when working with raw block devices.
-{{< /note >}}
+`PersistentVolumeClaims` will only be bound to `PersistentVolumes` with the same `volumeMode.`
```
Comment (msau42) [inline content/en/docs/concepts/storage/persistent-volumes.md]: Maybe one sentence is too small for an entire section. Would it make sense to merge it with the above section?

Comment (wongma7) [reply content/en/docs/concepts/storage/persistent-volumes.md]: yes, moved to some place above

File: content/en/docs/concepts/storage/persistent-volumes.md (hunk 2 above)
Comment (tengqm) [reply content/en/docs/concepts/storage/persistent-volumes.md]: This should be updated because the pr is targeting dev-1.13 branch.

**Pull request merged**